src/parameterizations/lateral/MOM_internal_tides.F90

```diff
@@ -2324,12 +2324,14 @@ subroutine internal_tides_init(Time, G, GV, US, param_file, diag, CS)
   call get_param(param_file, mdl, "REFL_ANGLE_FILE", refl_angle_file, &
                "The path to the file containing the local angle of "//&
                "the coastline/ridge/shelf with respect to the equator.", &
-               fail_if_missing=.false.)
+               fail_if_missing=.false., default='')
   filename = trim(CS%inputdir) // trim(refl_angle_file)
   call log_param(param_file, mdl, "INPUTDIR/REFL_ANGLE_FILE", filename)
   allocate(CS%refl_angle(isd:ied,jsd:jed)) ; CS%refl_angle(:,:) = CS%nullangle
-  call MOM_read_data(filename, 'refl_angle', CS%refl_angle, &
-                     G%domain, timelevel=1)
+  if (file_exists(filename, G%domain)) then
```

Comment:
I think that we need to think about how to handle the case where the input file does not exist both here and in the other two changes in this file. If the file is not specified (i.e., the default of ' ') silently not doing anything is probably the right answer, but if a name is given but the file does not exist there should either be a warning message or a fatal error.

Reply:
Agreed and fixed in new commit, also move the call to log_param. No need to log an empty parameter, right?